content/docs/1.4/scalers/azure-event-hub.md

```diff
@@ -4,7 +4,6 @@ layout = "scaler"
 availability = "v1.0+"
 maintainer = "Community"
 description = "Scale applications based on Azure Event Hubs."
-notice = "As of now, the Event Hub scaler only supports reading checkpoints from Blob Storage, as well as scaling only Event Hub applications written in C#, Java, Python or created with Azure Functions."
```

Comment:
Can you remove this in v2.3 instead please?

Reply:
oh, damn sry. done

Reply:
No problem, thanks!